R/Rules-methods.R

```diff
@@ -800,21 +800,24 @@ setMethod(
     }
 
     # extract dose names as these get lost if only one dose available
```

Comment:
yeah I meant all comments please in sentence style, e.g. also here
```suggestion
    # Extract dose names as these get lost if only one dose is available.
```
and similar all other comments in your class/methods/functions :-) thank you!

Reply:
Changes done.